docs/methods/handle-sign-in.md

```diff
@@ -0,0 +1,35 @@
+# Handle sign-in
+
+Build the sign-in callback page for your web app. You should [configure sign-in](https://www.transposit.com/docs/building/js-sdk/) before calling this method.
```

Comment:
nit: I wonder if we should hard wrap long lines. Upside: it makes it much easier to read in some editors. Downside: it could lead to more git churn when we change a paragraph and need to reformat it.

Reply:
By hard-wrapping, you mean putting in our own newlines? I am against this for now. We could revisit it if it's actually a problem.

Reply:
I feel fine with this.